### DISCUSSION TITLE
Exception Thrown: No module named 'discord'

### DISCUSSION THREAD

**Discussion by n-ooham** [Rapptz/discord.py#5825] [Q&A] [answered]
I've spent so many hours trying to fix this error, it happens every time 'import discord' is ran.

**Comment by Phxntxm** [comment, marked as answer]
This error at its simplicity means that the library isn't installed. If you are certain you have installed it but still receive this then it means you installed it into a different *environment* than you are running from. For example if you have 3 different python versions installed, this is 3 different environments... if you install to 1 version then it is not installed to the others.

Additionally you can have virtual environments, and some IDEs such as pycharm use one automatically. If you are running from pycharm then you either need to change your interpreter to use the same one you're installing to, install into pycharm's virtual environment, or stop running from pycharm and run from command line instead.

Regardless, this isn't an issue with the library however and github issues are not really the place to ask for this kind of support. If you want further help with this then you can ask in the [official support server](https://discord.gg/dpy)
